src/ApiIncrementalChecker.ts

```diff
@@ -160,4 +164,21 @@ export class ApiIncrementalChecker implements IncrementalCheckerInterface {
       allLints.map(this.createNormalizedMessageFromRuleFailure)
     );
   }
+
+  private emitFiles() {
+    try {
+      const program = this.tsIncrementalCompiler.getProgram();
+
+      if (!program) {
+        throw new Error(
+          'Program (a ts.Program object) in emitFiles() for ApiIncrementalChecker.ts not found'
+        );
+      }
+
+      program.emit(); // called here but the emission of files is in fact controlled by the boolean enableEmitFiles inside writeFile of CompilerHost
+    } catch (error) {
+      console.log('Error inside emitFiles() of ApiIncrementalChecker');
```

Comment:
these should possibly use the `this.logger` rather than `console`?

Reply:
Hi @johnnyreilly . You are probably right, but there is no "this.logger" inside "ApiIncrementalChecker.ts". How do you suggest to transfer this object from "index.ts" to "service.ts" (that, in his turn, will be transferring it to "ApiIncrementalChecker.ts" through its constructor)?